src/components/Header.jsx

```diff
@@ -1,18 +1,22 @@
 import React from 'react';
 import { NavLink } from 'react-router-dom';
 
+import '../styles/header/header.styl';
 
 const Header = () => (
-  <header>
-    <nav>
-      <ul>
-        <li><NavLink to="/" activeClassName="active">New snippet</NavLink></li>
-        <li><NavLink to="/recent" activeClassName="active">Recent snippets</NavLink></li>
-        <li><NavLink to="/upload" activeClassName="active">Upload snippet</NavLink></li>
-        <li><NavLink to="/about" activeClassName="active">About</NavLink></li>
-        <li><NavLink to="/sign-in" activeClassName="active">Sign in</NavLink></li>
-      </ul>
-    </nav>
+  <header className="header" key="header">
+    <span className="header-logo" />
+    <div className="header-wrapper">
```

Comment:
Do we really need so many wrappers? o_O

Reply:
There are three main parts on header, and to position them accordingly to design it's a smallest amount of wrappers